humble/distribution.yaml

```diff
@@ -1233,6 +1233,24 @@ repositories:
       url: https://github.com/eclipse-cyclonedds/cyclonedds.git
       version: releases/0.10.x
     status: maintained
+  data_tamer:
+    doc:
+      type: git
+      url: https://github.com/facontidavide/data_tamer.git
+      version: main
+    release:
+      packages:
+      - data_tamer
+      - data_tamer_msgs
+      tags:
+        release: release/humble/{package}/{version}
+      url: https://github.com/facontidavide/data_tamer-release.git
```

Comment:
While this is okay for `humble`, for `rolling` you will need to use ros2-gbp.  Are you sure you want to release here and potentially shift later or just go ahead and do ros2-gbp?

Reply:
I will be happy to release on **ros2-gbp**.
Submitted a request here: https://github.com/ros2-gbp/ros2-gbp-github-org/issues/389

Reply:
I will probably close this, since I want to release 0.9.0 soon, but let's prepare the release repo :mechanical_arm: